lib/health_check/custom_health_checks.rb

```diff
@@ -0,0 +1,53 @@
+module HealthCheck
+  class CustomHealthChecks
+
+    def self.check_sidekiq_redis
+      Sidekiq.redis { |r| "" if r.ping == "PONG" }
+    rescue Exception => e
+       create_error 'sidekiq-redis', e.message
+    end
+
+    def self.check_redis
+      "" if Redis.new.ping
+    rescue Exception => e
+       create_error 'redis', e.message
+    end
+
+    def self.check_s3
+      return create_error 's3', "Could not connect to aws" if aws_s3_client.nil?
+      HealthCheck.buckets.each do |bucket|
+        aws_s3_client.put_object(bucket: bucket, key: 'FOO', body: 'BAR')
```

Comment:
I know this ah gem, but this health_check is forked from public gem, and to make PR to public gem I can't use ah private gems. If they merge our PR, we can use public gem since then, but until then we need to use our fork :)